rubocop/missing_keys_on_shared_area.rb

```diff
@@ -0,0 +1,82 @@
+require 'rubocop'
+
+module RuboCop
+  module Lint
+    class MissingKeysOnSharedArea < RuboCop::Cop::Cop
+      MISSING_KEYS_MSG = "There are missing keys on the shared area. Check constants provided in 'SharedValues' or keys provided in 'output' method in the action's code".freeze
```

Comment:
It’d be nice to point to which keys are specifically missing, e.g. output `Found %<key>s In SharedValues but not in output method. Keys in the output method: %<list>s`

Reply:
Good idea 💪 Fixed with the newest commit 🙇